libs/tools/barista/src/transform.ts

```diff
@@ -300,3 +346,20 @@ export function internalContentTransformerFactory(
     return transformed;
   };
 }
+
+/** Checks whether a URL is relative  */
+function isRelativeUrl(href: string): boolean {
```

Comment:
I would like to add a quick sidenote on terminonlogy here.

_Relative Links_
Urls that either start without a slash or even dots in their href. They start resolving the URL from the pathname they are currently on: i.e. On the page `/components/button/` a relative link like `../` would be completely valid, and point to `/components/`;

_Absolute links_ 
are links that start with a slash, they start their navigation at the hostname level. That is why a link on the page `/components/button/` that looks like this `/patterns/guidelines` will end up on the barista.dynatrace.com/patterns/guidelines. 

_Fully qualified links_
Also include their hostname, i.e. `https://barista.dynatrace.com/components/button`.
Something like this, would also be a fully qualified link, but without a protocol specification (which means it will take the protocol of the page that you are currently on) `//barista.dynatrace.com/components/button`

--- 

The regex in this function checks if the url is fully qualified, but does not check if it is a relative path or not. 
If you expand the matcher in the demo to global and multiline (which will make it highlight the other test matches, you will see that it does not differentiate between relative and absolute links.
https://regex101.com/r/nzwuR4/3

Reply:
Unresolved this comment, because it is not fixed.